docs/requirements.md

```diff
@@ -0,0 +1,61 @@
+At its heart ServiceControl and ServicePulse provide operational support for systems built on the Particular Service Platform. They do this by:
+
+- monitoring system health
+- diagnosing and treating errors
+
+In this context, system refers to NServiceBus endpoints and communication with third-party services.
+
+ServiceControl should provide the following features:
+
+- process and store the contents of the error queue
+- replay messages from the error queue
+- process and store the contents of the audit queue
+- monitor the availability of endpoints
+- generate and collect saga state changes
+- provide an API for monitoring user-defined conditions
+- provide integration hooks for developers to consume
+- search for messages
+- export failed message data for external analysis
+- report on the contents of the timeout manager
+- allow user to provide metadata about messages for operational support
+- allow user to easily provide feedback and usage data and diagnostics for the platform
+- monitor the health of SC itself
+
+Implicit in these requirements is that the Service Platform should:
```

Comment:
We mentioned before that the list above are features, not requirements.

Could we treat the list below as additional features instead of requirements?